R/score_forecasts.R

```diff
@@ -0,0 +1,97 @@
+#' Score forecasts
+#'
+#' @param forecasts required data.frame with forecasts in the format returned
+#' by load_forecasts
+#' @param truths required data.frame with forecasts in the format returned
+#' by load_truth
+#' @param scores character vector of scores to calculate
+#' @param return_format string: "long" returns long format with a column for
+#' "score_name" and a column for "score_value"; "wide" returns wide format with
+#' a separate column for each score.
+#'
+#' @return data.frame with columns model, forecast_date, location,
```

Comment:
I'm not sure if returning a `truth_source` column would be helpful?

Reply:
Yeah, probably not... since users would pass in the truth dataframe the source is obviously the dataframe that they pass in.